pkg/scheduler/algorithm/priorities/interpod_affinity_test.go

```diff
@@ -612,3 +629,57 @@ func TestHardPodAffinitySymmetricWeight(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkInterPodAffinityPriority(b *testing.B) {
```

Comment:
should we move this to https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/scheduler_bench_test.go

Reply:
We should come up with a common library and make it consumed by both UT and Perf/Integration tests.

FYI: @draveness